test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -548,4 +558,75 @@ UCS_TEST_P(test_uct_cm_sockaddr, cm_server_reject)
                 (TEST_CM_STATE_SERVER_CONNECTED | TEST_CM_STATE_CLIENT_CONNECTED)));
 }
 
+
 UCT_INSTANTIATE_SOCKADDR_TEST_CASE(test_uct_cm_sockaddr)
+
+
+class test_uct_cm_sockaddr_multiple_cms : public test_uct_cm_sockaddr {
+public:
+    void init() {
+        ucs_status_t status;
+
+        test_uct_cm_sockaddr::init();
+
+        status = ucs_async_context_create(UCS_ASYNC_MODE_THREAD_SPINLOCK, &m_async);
+        ASSERT_UCS_OK(status);
+
+        UCS_TEST_CREATE_HANDLE(uct_worker_h, m_worker_test, uct_worker_destroy,
+                               uct_worker_create, m_async,
+                               UCS_THREAD_MODE_SINGLE)
+
+        UCS_TEST_CREATE_HANDLE(uct_cm_h, m_cm_test, uct_cm_close,
+                               uct_cm_open, GetParam()->component,
+                               m_worker_test);
+    }
+
+    void cleanup() {
+        test_uct_cm_sockaddr::cleanup();
+        ucs_async_context_destroy(m_async);
+    }
+
+    void server_accept(entity *server, uct_conn_request_h conn_request,
+                       uct_ep_server_connect_cb_t connect_cb,
+                       uct_ep_disconnect_cb_t disconnect_cb,
+                       void *user_data)
+    {
+        server->accept(m_cm_test, conn_request, connect_cb, disconnect_cb,
+                       user_data);
+    }
+
+protected:
+    ucs::handle<uct_worker_h> m_worker_test;
```

Comment:
"m_worker_test" -> "m_worker"
"m_cm_test" -> "m_cm"
"_test" makes me think that this a test instance

Reply:
to avoid confusion, i prefer not to have the same names as the base class (uct_test).

Reply:
i see, then probably m_test_worker?

Reply:
ok